readme.md

```diff
@@ -789,6 +877,7 @@ $ nox -s tests -- tests/test_report_switch_firmware.py
 - Added the ability to fully track device slot and port assignments.
 - Mountain hardware (CMM, CEC) is now being generated in the switch configurations.
 - Fixed multiple templates to match what is on the Aruba switch, these include, UANs, Loopbacks, VLAN interfaces, ACLs.
+- Added an `--override` option to `canu generate switch config` and `canu generate network config`, this allows users to ignore custom configuration so CANU does not overwrite it.
```

Comment:
this goes in "unreleased", not a release.  unreleased == develop and main.  once tagged then we move changes to released/tag.  good idea on the change date even for unreleased.  keep that per changeset.